test/regress/regress-2136.txt

```diff
@@ -0,0 +1,11 @@
+;;; TOOL: run-spec-wasm2c
+;;; ARGS: --debug-names
+(module
+ (func $B0 (export "$B0") (param $B0 i32)
+       (block
+	(br 0))))
+
+(assert_return (invoke "$B0" (i32.const 5)))
+(;; STDOUT ;;;
+1/1 tests passed.
```

Comment:
Once again I kind of which the full output was shown here, so that I could see that the change looks like in terms of its effect on the output.  IIRC we have an outstanding change to make that happen so it doesn't need to happen here.

Reply:
We have the three output tests (add.txt, minimal.txt, hello.txt) but that's it right now -- I don't think we have anything pending right now. This PR doesn't change the output (except in cases of name collision) so it won't affect those.

Happy to make more output tests if you want. (I mean we could lock in the full output of every spec test if you want, but, it would probably mean a lot of noise whenever we change the output.)